### DISCUSSION TITLE
Sent coins from Chia wallet, nothing recieved.

### DISCUSSION THREAD

**Discussion by valentosnik** [Chia-Network/chia-blockchain#9380] [Support]
Hello,

I send 2 day ago 1.3 xch from my chia wallet to my wallet at gate.io. Nothing recieved untill now. They need TXID/Transaction ID, but this information is not available at Chia wallet. Where is my money, what can I do?

I tryed to do the same with smaller value and it worked! What can be a problem. There is no failure in wallet number.

Please help.

Best regards,
Valentin.

**Comment by Ryko** [comment]
Hi, same problem for me with gate.io exchange... they ask me for the TXID but can't find it anywhere... please help!

**Comment by valentosnik** [comment]
I found how to find transaction ID. You just go to:
C:\Users\youruser\AppData\Local\chia-blockchain\app-1.2.6\resources\app.asar.unpacked\daemon
Open cmd in it and put this command: chia wallet get_transactions

Then you can find it. But now gate.io doesnt react on it...

**Comment by valentosnik** [comment]
And here you can find your transaction:

https://xchscan.com/
